packages/harpguru-core/src/components/harp-face/hooks/use-styles.ts

```diff
@@ -2,22 +2,30 @@ import { useGlobal } from 'reactn'
 import { StyleSheet } from 'react-native'
 import type { ViewStyle } from 'react-native'
 
-import { useSizes } from '../../../hooks'
-
-import { useOctaveColumnGroups } from './use-octave-column-groups'
+import { useSizes, useOctaveColumnGroups } from '../../../hooks'
 
 type HarpFaceStyles = {
   readonly face: ViewStyle
 }
 
-export const useStyles = (): HarpFaceStyles => {
+export const useStyles = (
+  harpfaceIndex: 'harpface1' | 'harpface2'
+): HarpFaceStyles => {
   const { dynamicSizes } = useSizes()
   const { columnWidth, rowHeight, fragmentGutter } = dynamicSizes
 
   const [layoutFacts] = useGlobal('layoutFacts')
-  const { harpfaceRows, harpfaceColumns } = layoutFacts
-
-  const octaveColumnGroups = useOctaveColumnGroups()
+  const {
+    [harpfaceIndex === 'harpface1' ? 0 : 1]: { harpfaceRows, harpfaceColumns },
+  } = layoutFacts
+
+  // TODO: Should we perhaps just make useOctaveColumnGroups always use
```

Comment:
Yes, definitely for now. It's a clear indication of design intent. I have no usecase for fragmenting by the second harpfaces octave boundaries.

Reply:
Done